Sources/KlaviyoCore/Networking/NetworkSession.swift

```diff
@@ -35,10 +35,39 @@ public struct NetworkSession {
 
     public static let networkTimeout: UInt64 = 10_000_000_000 // in nanoseconds (10 seconds)
 
+    static func getPluginConfiguration(bundle: Bundle = .main) -> (name: String, version: String)? {
```

Comment:
```suggestion
    private static func getPluginConfiguration(bundle: Bundle = .main) -> (name: String, version: String)? {
```
looks like this can be private. Unfortunately you'll need to revise your tests if you make it private, but the updates shouldn't be too difficult.